future_challenges_for_RSEng.tex

```diff
@@ -155,19 +155,6 @@ \subsection{Education, and changes in the educational system}
 \subsection{Impact of AI and automation}
 Is it better than cheap PhDs?
 
-\subsection{How have academic Institutions transformed by then? ( Point 4 in the issue )}
-- Emerging structures:
-  - Universities/Institutes with central pools of RSEs, each specializing in data management, visualization, HPC, etc.
-  - Potential for closer collaboration between domain scientists and RSEs to share workload effectively.
-
-- Challenges:
-  - Budget and administrative hurdles to creating permanent, well-funded RSE teams.
```

Comment:
The budget might be part of the solution: pooling solutions can be more efficient than others. However, the administrative hurdles are the really hard ones.

Reply:
so you do not think, that the current "What do we want to have achieved, and what is still a problem" describes these parts?
On the other hand, I didn't want to reproduce the RSE groups paper....